src/main/scala/com/datastax/sparkstress/ConnectHelper.scala

```diff
@@ -0,0 +1,16 @@
+package com.datastax.sparkstress
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.SparkSession
+
+object ConnectHelper {
+  def getSparkSession(conf: SparkConf = DefaultConfig): SparkSession = {
+    SparkSession.builder().config(conf).getOrCreate()
+  }
+
+  val DefaultConfig = new SparkConf()
+    .setAppName("Spark Stress Test")
+    .setMaster("local[*]")
+    .set("spark.hadoop.fs.dsefs.impl", "com.datastax.bdp.fs.hadoop.DseFileSystem") // without we get: 'java.io.IOException: No FileSystem for scheme: dsefs'
+
+}
```

Comment:
Why did we move the location of this file?

Reply:
Previously this was sitting in the DSE build directory and we also had one in the Maven directory. Since the code was identical i just consolidated and moved it to the shared location